packages/control-property-editor/src/panels/outline/Tree.tsx

```diff
@@ -358,43 +359,66 @@ export const Tree = (): ReactElement => {
                 aria-hidden
                 className={`${selectNode} tree-row ${focusEditable}`}
                 onClick={(): void => onSelectHeader(groupHeaderProps?.group)}>
-                <span
-                    style={{ paddingLeft: paddingValue }}
-                    data-testid="tooltip-container"
-                    className={`tree-cell ${isExtensionPoint ? 'tooltip-container' : ''}`}
-                    onContextMenu={(e) => isExtensionPoint && handleOpenTooltip(e, tooltipId)}>
-                    {groupHeaderProps?.group?.count !== 0 && (
-                        <Icon
-                            className={`${chevronTransform}`}
-                            iconName={UiIcons.Chevron}
-                            onClick={(event) => {
-                                onToggleCollapse(groupHeaderProps);
-                                event.stopPropagation();
+                {scenario !== 'ADAPTATION_PROJECT' ? (
```

Comment:
Could you please use the `scenario` const from `@sap-ux-private/control-property-editor-common`?

Reply:
This one here should not be a constant since it is coming from redux store.

Reply:
Right 👍